src/BenchmarkDotNet/Engines/StoppingCriteria/StoppingResolution.cs

```diff
@@ -0,0 +1,21 @@
+using JetBrains.Annotations;
+
+namespace BenchmarkDotNet.Engines
+{
+    public struct StoppingResolution
```

Comment:
nit: I think that `StoppingResult` would be a better name, we already use the `Result` convention in few places